### PR TITLE
Remove Unneeded Delays

### PR DESCRIPTION
**Pull request by RoyalSix**
#### Describe what your pull request addresses (Please include relevant issue numbers):
-

#### Please include detailed Test instructions for your pull request:
-

#### Standard Test Instructions for PR Review Process:

- [ ] Double check unit tests that have been written
- [ ] Check for documentation for code changes
- [ ] Check that there are not inadvertent commits to tC Apps when reviewing a tC Core PR
- [ ] Checkout the branch locally and ensure that app runs as expected
  - [ ] Ensure tests pass
  - [ ] Open and watch the console for errors
  - [ ] Make sure all actions perform as expected
  - [ ] Import and Load a new Project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Switch project to an existing project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Next time reverse the order of importing after loading an existing project
- [ ] Reviewer should double check the DoD in the ISSUE, including the “spirit” of the story
- [ ] Ask Ben or Koz if you have any concerns about implementation (especially UI related)

---
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword-dev/translationcore/6394)

### CODE REVIEW

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (09bf118)](https://tc-builds.door43.org/build/feature-jay-6282-4/translationCore-linux-x64-2.0.0-09bf118.deb) is ready for download.
[macOS build 2.0.0 (09bf118)](https://tc-builds.door43.org/build/feature-jay-6282-4/translationCore-macos-x64-2.0.0-09bf118.dmg) is ready for download.
[Windows x64 build 2.0.0 (09bf118)](https://tc-builds.door43.org/build/feature-jay-6282-4/translationCore-win-x64-2.0.0-09bf118.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (09bf118)](https://tc-builds.door43.org/build/feature-jay-6282-4/translationCore-win-x32-2.0.0-09bf118.setup.exe) is ready for download.

File: src/js/actions/VerseEditActions.js [hunk 1]
```diff
@@ -274,9 +271,8 @@ export const updateVerseEditStatesAndCheckAlignments = (verseEdit, contextIdWith
     }
     dispatch(AlertModalActions.closeAlertDialog());
     if (showSelectionInvalidated || showAlignmentsInvalidated) {
-      await delay(250);
       dispatch(showInvalidatedWarnings(showSelectionInvalidated, showAlignmentsInvalidated));
-      await delay(1000);
+      //TODO: add await here
```
Comment (mannycolon) [inline src/js/actions/VerseEditActions.js]: Do we need to add back the delay here?

Comment (RoyalSix) [reply src/js/actions/VerseEditActions.js]: Removed. It is no longer needed

**Review by mannycolon — APPROVED** [review]

**Comment by mannycolon** [comment]
Waiting on travis

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (be2353c)](https://tc-builds.door43.org/build/feature-jay-6282-4/translationCore-linux-x64-2.0.0-be2353c.deb) is ready for download.
[macOS build 2.0.0 (be2353c)](https://tc-builds.door43.org/build/feature-jay-6282-4/translationCore-macos-x64-2.0.0-be2353c.dmg) is ready for download.
[Windows x64 build 2.0.0 (be2353c)](https://tc-builds.door43.org/build/feature-jay-6282-4/translationCore-win-x64-2.0.0-be2353c.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (be2353c)](https://tc-builds.door43.org/build/feature-jay-6282-4/translationCore-win-x32-2.0.0-be2353c.setup.exe) is ready for download.

**Pull request merged**